### ISSUE TITLE
Filecoin should support legacy Ethereum transactions

### ISSUE THREAD

**Issue by aarshkshah1992** [filecoin-project/lotus#11859]
### Checklist

- [X] This is **not** brainstorming ideas. If you have an idea you'd like to discuss, please open a new discussion on [the lotus forum](https://github.com/filecoin-project/lotus/discussions/categories/ideas) and select the category as `Ideas`.
- [X] I **have** a specific, actionable, and well motivated feature request to propose.

### Lotus component

- [X] lotus daemon - chain sync
- [X] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### What is the motivation behind this feature request? Is your feature request related to a problem? Please describe.

This issue has been created to add support for legacy transactions to Filecoin so that contracts and user transactions are not scoped to a specific chain. Specifically, this issue tracks FIP discussion at https://github.com/filecoin-project/FIPs/discussions/962.

### Describe the solution you'd like

Option #3 on https://github.com/filecoin-project/FIPs/discussions/962 

"We could also prepend a transaction type byte to the signature field (for Ethereum transactions only). IMO, this is the best solution, but it may break any tooling manually signing/validating messages from Ethereum accounts. On the other hand, I'm doubtful any tooling like this exists, so we're probably fine."

**Comment by aarshkshah1992** [comment]
@Stebalien 

Just to clarify,  when we support legacy transactions - do we mean
Transactions **_without_** Chain ID OR
Transactions without ChainID AND without any of the EIP-1599 gas stuff params ?

**Comment by Stebalien** [comment]
Hm. It's going to have to be the latter, I think (needs to be bit for bit compatible). I _think_ we need to set the `GasPremium` to `GasFeeCap` when translating from the EthTxn to the Filecoin message.

**Comment by Stebalien** [comment]
Yeah, from what I can tell, the premium is the gas fee. The FVM will subtract out whatever it needs to burn before giving the rest to the SP.

**Comment by rjan90** [comment]
Closing this ticket, since this has been implemented with:
- https://github.com/filecoin-project/lotus/pull/11969
- https://github.com/filecoin-project/lotus/pull/11970

And is planned to go live with the upgrade to NV23
